Libraries/LibGUI/ModelIndex.cpp

```diff
@@ -46,4 +46,14 @@ const LogStream& operator<<(const LogStream& stream, const ModelIndex& value)
     return stream << String::format("ModelIndex(%d,%d)", value.row(), value.column());
 }
 
+} // namespace GUI
```

Comment:
We don't put these comments anywhere else, let's not put it here

Reply:
They do appear in some places in the code base. Personally, I like to add them when multiple namespaces are involved to make the code more readable.

Reply:
> They do appear in some places in the code base.

To be fair, the only ones that I found are in `AK/Format.h` and `AK/Format.cpp`, and were added by you :/

Reply:
> [`CodingStyles.md`](https://github.com/SerenityOS/serenity/blob/52c31bb743ce5baa6358f1db34c5341833b8cdf1/Documentation/CodingStyle.md)

That isn't "codebase", and they aren't comments (just text); the only **comments** of that kind in the code are by you.

> so I'll leave it in.

Ok, that's fine.

Reply:
@awesomekling What's your opinion on this?

Reply:
Let's just skip the `// namespace Foo`.